lib/python/test/KHI_growthRate/MainTest.py

```diff
@@ -0,0 +1,38 @@
+import sys
+from time import *
+
+lt = localtime()
+date = strftime("date: %d.%m.%Y", lt)
+timeOfDay = strftime("time: %H:%M:%S", lt)
+
+try:
+	sys.path.append(sys.argv[1])
+	from TestKHI import TestKHI
+	from ViewerKHI import ViewerKHI
+
+	field = ["total", "B_x", "B_y", "B_z", "E_x", "E_y", "E_z"]
+
+	paramDir = sys.argv[2]
+	simDir = sys.argv[3]
```

Comment:
Which files must be located in `simDir`?
If you need the openPMD output maybe the prefix path + filename would be better. e.g. `simOutpur/opePMD/simData`.
The openPMD file names and folder can be chosen by the user, therefore we should allow to specify it.
IMO it would be useful to use the python argument parses instead of using unnamed fixed position parameters for the testing tool, see https://github.com/ComputationalRadiationPhysics/picongpu/blob/d9ab3532a2b0d10e752b5b701ec14254f486d176/src/tools/bin/plot_chargeConservation.py#L184